.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: php
 php:
-  - '5.5'
-  - '5.6'
+  - '7.1'
   - '7.2'
-  - nightly
```

Comment:
Why aren't we compatible with upcoming versions? Is it due to hard dependencies in libs? Can we at least have something like `7.* (latest)`?

Reply:
added `nightly` back